ux.symfony.com/templates/ux_packages/toolkit.html.twig

```diff
@@ -0,0 +1,185 @@
+                        <twig:CardDescription>Symfony is a set of reusable PHP components...</twig:CardDescription>
+                    </twig:CardHeader>
+                    <twig:CardContent>
+                        ... and a PHP framework for web projects
+                    </twig:CardContent>
+                    <twig:CardFooter>
+                        <twig:Button primary class="w-full">Visit symfony.com</twig:Button>
+                    </twig:CardFooter>
+                </twig:Card>
+            {% endblock %}
+        </div>
+    </div>
+
+    <div class="row my-3">
+        <div class="col-6">
+            <twig:CodeBlock
+                filename="templates/ux_packages/toolkit.html.twig"
+                :showFilename="false"
+                targetTwigBlock="avatar_example"
+                :showTwigExtends="false"
+                height="auto"
+            />
+        </div>
+        <div class="col-6">
+            {% block avatar_example %}
+                <twig:Avatar class="inline-block">
+                    <twig:AvatarImage src="https://github.com/symfony.png" alt="@symfony" />
+                    <twig:AvatarFallback>SF</twig:AvatarFallback>
+                </twig:Avatar>
+
+                <twig:Avatar class="inline-block">
+                    <twig:AvatarFallback class="bg-blue-400">SF</twig:AvatarFallback>
+                </twig:Avatar>
+            {% endblock %}
+        </div>
+    </div>
+{% endblock %}
+
+
+{% block package_install %}
+    {% set installOnlyInDevMode = true %}
+    {{ parent() }}
+    <section class="container-fluid container-xxl px-4 pt-4 px-md-5 pt-md-5">
+        <p>
+            Then install your components with the following command:
+        </p>
+
+        {% component Terminal with {bottomPadding: 20} %}
+            {% block content %}
+                php bin/console ux:toolkit:install badge
+                php bin/console ux:toolkit:install button
+                php bin/console ux:toolkit:install card
+                php bin/console ux:toolkit:install ...
+            {% endblock %}
+        {% endcomponent %}
+
+        <p class="mt-4">
+            Official components require tailwindcss to be installed in your project. Visit the <a class="text-underline" target="_blank" href="https://tailwindcss.com/docs/installation" target="_blank">Tailwind CSS documentation</a> for more information.
+            Components are based on <a class="text-underline" target="_blank" href="https://ui.shadcn.com/" >shadcdn/ui</a> design system.
+        </p>
+        
+        <p class="mt-4">
+            If you use component library using <code>html_cva()</code> or <code>tailwind_merge()</code> functions, remember to install them with:
+        </p>
+
+        {% component Terminal with {bottomPadding: 20} %}
+            {% block content %}
+                composer require --dev tales-from-a-dev/twig-tailwind-extra twig/extra-bundle
```

Comment:
Since we rely on `html_cva`, we must tell the user to install `twig/html-extra:^3.12.0` as well